ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/jackson/mixin/PaymentIntentionMixin.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.cmc.ccd.jackson.mixin;
+
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import uk.gov.hmcts.cmc.domain.models.PaymentOption;
+import uk.gov.hmcts.cmc.domain.models.RepaymentPlan;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+public abstract class PaymentIntentionMixin {
+
+    abstract PaymentOption getPaymentOption();
```

Comment:
why do we need these methods which doesnt have annotations ? can we remove these

Reply:
indeed